lib/watir-webdriver/elements/element.rb

```diff
@@ -217,9 +221,11 @@ def drag_and_drop_by(right_by, down_by)
       assert_exists
       assert_has_input_devices_for :drag_and_drop_by
 
-      driver.action.
-             drag_and_drop_by(@element, right_by, down_by).
-             perform
+      element_call do
+        driver.action.
```

Comment:
So the one before definitely had the period in the wrong spot (bottom line instead of top). This one is ok, though, or is the issue something else? Thanks.